crawler/file_processing.py

```diff
@@ -193,9 +194,17 @@ def download_csv_files(self) -> None:
             logger.debug(f"ls {self.config.SFTP_HOST}/{sftp_root_read}")
             logger.debug(sftp.listdir(sftp_root_read))
 
-            # download all plate map files
+            # download recent plate map files
             logger.info("Downloading plate map files")
-            sftp.get_d(self.centre_config["sftp_root_read"], self.get_download_dir())
+            for csv_file in sftp.listdir(sftp_root_read):
+                sftp.chdir(sftp_root_read)
+                timestamp = sftp.stat(csv_file).st_mtime  # get timestamp of file
+                createtime = datetime.fromtimestamp(timestamp)
```

Comment:
Is the creation time the one we want?  I would have thought the modified time would have been better.  I don't know much about the creation of the files by the lighthouses, but is there a chance they modify the contents of an existing file and we don't pick up that change?  Or do they always replace files even when they end up using the same filename?

Reply:
It seems like this is actually already the modified time, so it would be good to rename this variable `modified_time`